test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -255,6 +256,71 @@ public void SimpleConstructorsAndExtensions()
             CheckSameValues(data6, data7);
             CheckSameValues(data6, data8);
 
+            // Tests for LogMeanVariance
+            var est9 = new NormalizingEstimator(Env, NormalizingEstimator.NormalizationMode.LogMeanVariance, ("float4", "float4"));
```

Comment:
> "float4" [](start = 119, length = 9)

I'd prefer in *new* tests to avoid using non-public API. Is that impossible in this case for some reason?

Reply:
I carved out a separate test `NormalizerExperimentalExtensions`  to exercise the experimental APIs

---
In reply to: [268185782](https://github.com/dotnet/machinelearning/pull/3052#discussion_r268185782) [](ancestors = 268185782)